packages/cli/src/services/prompt-processors/atFileProcessor.ts

```diff
@@ -68,8 +69,9 @@ export class AtFileProcessor implements IPromptProcessor {
             error instanceof Error ? error.message : String(error);
           const uiMessage = `Failed to inject content for '@{${pathStr}}': ${message}`;
 
-          console.error(
-            `[AtFileProcessor] ${uiMessage}. Leaving placeholder in prompt.`,
+          // `context.invocation` should always be present at this point.
```

Comment:
nit: make this polished and provide a fallback if context.invocation is not known or use the `!` syntax if you are confident it is known.